### ISSUE TITLE
[Bug] Error: Invalid access of discarded (invalid, deleted) uWS.HttpResponse/SSLHttpResponse.

### ISSUE THREAD

**Issue by masfahru** [kartikk221/hyper-express#184]
## Error
Error happening when client is aborting request

```bash
Error: Invalid access of discarded (invalid, deleted) uWS.HttpResponse/SSLHttpResponse.
    at Response.send (/home/fahru/Kerja/test-hyper/node_modules/hyper-express/src/components/http/Response.js:491:39)
    at Response.json (/home/fahru/Kerja/test-hyper/node_modules/hyper-express/src/components/http/Response.js:698:34)
```

## Step to reproduce
1. Init Nodejs app, install Hyper-Express 6.8.0

2. Create index.js, add below code

```javascript
import { Server } from 'hyper-express'

const delay = (t, val) => {
  return new Promise(function(resolve) {
      setTimeout(function() {
          resolve(val);
      }, t);
  });
}

const someAsyncTask = async () => {
  return delay(2000, 'Hey wait for me!');
}

const webserver = new Server()

webserver.get('/', async (_request, response) => {
  const r = await someAsyncTask();
  response.json({ message: r })
})

webserver.listen(3000)
```

3. Start the server using `node index.js`
4. `curl "http://127.0.0.1:3000/"` and abort the request before it return response

The error is happened in version 6.8.0
![Screenshot_20230727_104721](https://github.com/kartikk221/hyper-express/assets/92277627/e6ab7660-35d9-4308-8796-2b9f717f5342)

Tested in version 6.7.1 and it's working fine
![image](https://github.com/kartikk221/hyper-express/assets/92277627/dc2cd116-7721-4255-8e9c-8476ce6252e1)

Able to prevent the in version 6.8.0 if the response.json is wrapped using response atomic:
```javascript
webserver.get('/', async (_request, response) => {
  const r = await someAsyncTask();
  response.atomic(()=>{
    response.json({ message: r })
  })
})
```
![Screenshot_20230727_104811](https://github.com/kartikk221/hyper-express/assets/92277627/abd14ddb-4b32-4a21-9d5d-5c14a3299d8f)

**Comment by kartikk221** [comment]
This bug has been fixed in the recently releases `v6.8.1` update. Be sure to update with `npm` and let me know If that resolves your problem. Thanks!

**Comment by masfahru** [comment]
> This bug has been fixed in the recently released `v6.8.1` update. Be sure to update with `npm` and let me know If that resolves your problem. Thanks!

It's been solved, thank you !
